tools/build/makeBLOB_ROLES.raku

```diff
@@ -58,8 +58,9 @@ while @lines {
     say Q:to/SOURCE/.subst(/ '#' (\w+) '#' /, -> $/ { %mapper{$0} }, :g).chomp;
 
 my role #name#[::T] is repr('VMArray') is array_type(T) is implementation-detail {
-    method !push-List(str $action, ::?CLASS:D $to, \from) {
+    method !push-List(str $action, ::?CLASS:D \to, \from) {
```

Comment:
Does the part about `Blob`/`Buf` really belongs to this PR?

Reply:
It's a commit I've been hoping to get rid of, but `S03-operators/buf.t` wouldn't play nice with the other changes without. Doing the `Mu` type cache allows those to pass.